ifrau-client.js

```diff
@@ -0,0 +1,3 @@
+module.exports = import('ifrau').then(function(ifrau) {
+	return ifrau.Client;
```

Comment:
An alternative here would be to restrict this file to just importing ifrau and then fetch the client promise in `framed.js`. I don't have strong opinions here; I feel like we don't miss out on much by having this one line also not be linted. 🤷

Reply:
We don't have an export for _just_ the client?

Might've missed how that was going to work in the ifrau PR, but maintaining that feels important to me.

Also,

```suggestion
	return ifrau.SlimClient;
```

Reply:
So something like this works fine in ESM-land:
```import { SlimClient } from 'ifrau';```

Unfortunately there's no analogue with dynamic imports in CJS. We have to import the package as a default and then fetch its functions accordingly.

Reply:
But even that (without a tree-shaking bundler) is loading/downloading _everything_ to only use part of the functionality?

Like, maybe we want an import map (or `exports` in the ifrau package) so we can import just `ifrau/SlimClient` or the like?

Reply:
I could do that, yeah, but I suppose I don't entirely understand what we gain. The client and host themselves are very small and the majority of the ifrau code is in the `Port`, `PortWithRequests`, and `PortWithServices` classes, which are going to be required on import regardless because of how the inheritance structure is laid out. I feel like we don't get that much gain from just excluding `host.js` from a client import I guess, unless we look at restructuring ifrau to try to isolate dependencies in those port classes more.

Reply:
Or is the concern more that we want to make it explicit for consumers to use X and only X if they're a client, vs. Y and only Y if they're a host?

Reply:
(Either way I'm not opposed to doing this; I'm just trying to figure out whether there's more work that would be needed to accomplish what you're looking for.)

Reply:
Apparently committing the suggestion also resolves the conversation... Anyways, we chatted offline and I understand now. I neglected to think about the cost of importing the plugins as well. I'll split out these imports in ifrau and provide separate entrypoints for the host and the two types of clients. :)